platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -846,64 +890,38 @@ namespace platoon_strategic_ihp
             mob_op_cb_preparetojoin(msg);
         }
         // TODO: Place holder for prepare to depart
+
         // TODO: If needed (with large size platoons), add a queue for status messages
-        // INFO messages always processed, STATUS messages if saved in que
+        //       INFO messages always processed, STATUS messages if saved in que
     }
     
     void PlatoonStrategicIHPPlugin::mob_op_cb_standby(const cav_msgs::MobilityOperation& msg)
     {
         // In standby state, it will ignore operation message since it is not actively operating
+        ROS_DEBUG_STREAM("STANDBY state no further action on message from " << msg.m_header.sender_id);
     }
 
     // Handle STATUS operation message 
     void PlatoonStrategicIHPPlugin::mob_op_cb_candidatefollower(const cav_msgs::MobilityOperation& msg)
     {
-        std::string strategyParams = msg.strategy_params;
-        bool isPlatoonStatusMsg = (strategyParams.rfind(OPERATION_STATUS_TYPE, 0) == 0);
-        if(isPlatoonStatusMsg) 
-        {
-            mob_op_cb_STATUS(msg);
-            ROS_DEBUG_STREAM("Platoon STATUS operation message processed in candidatefollower state.");
-        }
-        else 
-        {
-            ROS_DEBUG_STREAM("Received a mobility operation message with params " << msg.strategy_params << " but ignored.");
-        }
+        ROS_DEBUG_STREAM("CANDIDATEFOLLOWER state no further action on message from " << msg.m_header.sender_id);
     }
 
     // Handle STATUS operation message 
     void PlatoonStrategicIHPPlugin::mob_op_cb_follower(const cav_msgs::MobilityOperation& msg)
     {
-        std::string strategyParams = msg.strategy_params;
-        bool isPlatoonStatusMsg = (strategyParams.rfind(OPERATION_STATUS_TYPE, 0) == 0);
+        //std::string strategyParams = msg.strategy_params;
+        //bool isPlatoonStatusMsg = (strategyParams.rfind(OPERATION_STATUS_TYPE, 0) == 0);
+
         // TODO: Place holder for prepare to depart
 
-        if(isPlatoonStatusMsg) 
-        {
-            mob_op_cb_STATUS(msg);
-            ROS_DEBUG_STREAM("Platoon STATUS operation message processed in follower state.");
-        }
-        // TODO: Place holder for prepare to depart (else if isDepart)
-        else 
-        {
-            ROS_DEBUG_STREAM("Received a mobility operation message with params " << msg.strategy_params << " but ignored.");
-        }
+        ROS_DEBUG_STREAM("FOLLOWER state no further action on message from " << msg.m_header.sender_id);
```

Comment:
I dont understand why the status is not processed here anymore?

Reply:
All it was doing was checking op type and calling mob_op_cb_STATUS() if appropriate.  That is now done by mob_op_cb() before this method is reached.  I seriously think it would be an improvement to remove all these mob_op_cb_*() methods that are now doing nothing but writing logs.  It was a nice idea in the beginning to have the flexibility for each of these to be broken out like this, but most of them are proving to be unneeded, and therefore just clutter.  Anyway, that can be considered another time.